### PR TITLE
Tolerate all taints in manifests

### PR DESCRIPTION
**Pull request by ozdanborne**
## Description

Tolerate all taints in manifests.

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by caseydavenport** [comment]
I will get to this after kubecon - it's not urgent?

**Comment by ozdanborne** [comment]
@caseydavenport not urgent at all.

File: master/getting-started/kubernetes/installation/hosted/calico.yaml [hunk 1]
```diff
@@ -287,6 +283,8 @@ spec:
       # it isn't governed by policy that would prevent it from working.
       hostNetwork: true
       serviceAccountName: calico-kube-controllers
+      tolerations:
+        - operator: "Exists"
```
Comment (caseydavenport) [inline master/getting-started/kubernetes/installation/hosted/calico.yaml]: Do we need "effect: NoSchedule" as well?

I think we probably should specify that.

**Comment by klausenbusk** [comment]
Any update on this?

**Comment by tmjd** [comment]
The changes in #1634 are very similar and I know work is ongoing there. Does what is in #1634 address your concerns @klausenbusk ?
(I think this PR can probably be closed with that other one in progress)
@heschlie, looks like you should be aware of this PR.

**Comment by klausenbusk** [comment]
> The changes in #1634 are very similar and I know work is ongoing there. Does what is in #1634 address your concerns @klausenbusk ?

I wasn't aware of #1634, thanks! I need a similar fix for `bootkube`, but they want to stick to upstream (calicio), so I'm waiting for upstream to implement the same changes. See: https://github.com/projectcalico/calico/pull/1634

**Comment by caseydavenport** [comment]
I think we should close this in favor of #1634

**Comment by ozdanborne** [comment]
Yep, closing in favor of #1634

**Pull request closed without merging**